pkg/network/tlsSecurityProfile_test.go

```diff
@@ -63,4 +63,24 @@ var _ = Describe("Testing TLS Security Profile", func() {
 			expectedMinTLSVersion: testCustomTLSProfileSpec.MinTLSVersion,
 		}),
 	)
+
+	Context("When selecting ciphers", func() {
+		It("should not generate duplicates", func() {
+			var profile = &ocpv1.TLSSecurityProfile{
+				Type: ocpv1.TLSProfileCustomType,
+				Custom: &ocpv1.CustomTLSProfile{
+					TLSProfileSpec: ocpv1.TLSProfileSpec{
+						Ciphers: []string{"foo", "foo", "bar"},
+					},
+				},
+				Intermediate: &ocpv1.IntermediateTLSProfile{},
+			}
+			var ciphers, _ = SelectCipherSuitesAndMinTLSVersion(profile)
+			for i, vi := range ciphers {
+				for j := i + 1; j < len(ciphers); j++ {
+					Expect(vi).ToNot(Equal(ciphers[j]))
+				}
+			}
```

Comment:
We should just compare with the exact slice using gomega matchers, dulication at test is worth it.
```suggestion
			ciphers, _ := SelectCipherSuitesAndMinTLSVersion(profile)
			Expect(ciphers).To(ConsistOf("foo", "bar"))
		
```

Reply:
I differ here, and unrolling loops in tests is not worth it.
In this specific case, the reason is that any time you modify/change/add or remove a cipher, you add a completely redundant task to update the test.

I do believe that this is unnecessary and unneeded maintenance, would you mind expanding on why it would be any better?

Reply:
ok, fair enough.